clang/include/clang/Basic/TargetInfo.h

```diff
@@ -1188,6 +1190,14 @@ class TargetInfo : public TransferrableTargetInfo,
       TiedOperand = N;
       // Don't copy Name or constraint string.
     }
+
+    // CC range can be set by target. SystemZ sets it to 4. It is 2 by default.
```

Comment:
Comment is wrong now (2 is no longer default).  Also, it's probably not necessary to specifically call out SystemZ here.